.gitignore

```diff
@@ -11,4 +11,7 @@ obj/
 *.nupkg
 .idea
 
-# End of https://www.gitignore.io/api/xamarinstudio
\ No newline at end of file
+# End of https://www.gitignore.io/api/xamarinstudio
+InteractiveAlert/.DS_Store
```

Comment:
I think you can remove "InteractiveAlert/.DS_Store", because ".DS_Store" works in folder recyrsive.

Reply:
Agreed